core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala

```diff
@@ -76,6 +78,9 @@ class SubmissionSupervisor(executionServiceCluster: ExecutionServiceCluster,
       }
   }
 
+  private def restartCounter(workspaceName: WorkspaceName, submissionId: UUID, cause: Throwable): Counter =
+    workspaceSubmissionMetricBuilder(workspaceName, submissionId).expand("cause", cause).asCounter("smaRestart")
```

Comment:
i think i'd prefer `monitorRestarted`, especially if the counter name gets tagged on the end of the metric